testing/kuttl/e2e-other-gke/cluster-migrate/04-errors.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: non-crunchy-cluster-0
```

Comment:
Don't quote me on this yet, but: I thought the way this would work is that each step # gets its own timeout, but it looks like if you have `00--delete` and `00-errors`, then both get the timeout length:

Delete's timeout: https://github.com/kudobuilder/kuttl/blob/63f81f8f6f40913631578b18fe3042ef26609aef/pkg/test/step.go#L155

Assert/error get run here with ctx with timeout:
https://github.com/kudobuilder/kuttl/blob/63f81f8f6f40913631578b18fe3042ef26609aef/pkg/test/utils/kubernetes.go#L1069-L1072

Reply:
> Do KUTTL error assertions wait for the entire step timeout?

Looking here, I think the answer is "no." When no `00-errors` match, `nil` bubbles up, and `"test step completed"`. KUTTL does not wait any time to find those.

📝 When we want to check that a field is not some value, we should be certain its object exists -- with an empty-ish `00-assert`, if need be.

- https://github.com/kudobuilder/kuttl/blob/63f81f8f6f40913631578b18fe3042ef26609aef/pkg/test/step.go#L438
- https://github.com/kudobuilder/kuttl/blob/63f81f8f6f40913631578b18fe3042ef26609aef/pkg/test/step.go#L385
- https://github.com/kudobuilder/kuttl/blob/63f81f8f6f40913631578b18fe3042ef26609aef/pkg/test/step.go#L310

Reply:
Followup on this: Chris and I talked about it, because I suspect I misread his first comment: `asserts` steps will wait the entire time _until_ the condition is matched and then exit early. (This has bitten us before when we try to delete a pod to regen it, and assert on it existing -- well, the pod that's being terminated still exists, which matches the assert condition.)

Likewise, `errors` steps will exit early if the condition is negatively matched. (So if you `errors` for a pod existing, as soon as that pod doesn't exist, that satisfies the condition and exits. But if the pod did exist, we would wait the whole timeout length.)